README.md

```diff
@@ -182,6 +182,18 @@ FoundationRailsHelper.configure do |config|
 end
 ```
 
+### Ignored Flash Keys
+The flash helper assumes all flash entries are user-viewable messages.
+To exclude flash entries which are used for storing state
+(e.g. [Devise's `:timedout` flash](https://github.com/plataformatec/devise/issues/1777))
+you can specify a blacklist of keys to ignore with the **ignored_flash_keys** config option:
+```ruby
+FoundationRailsHelper.configure do |config|
```

Comment:
Adding another example `FoundationRailsHelper.configure` block seems redundant and possibly confusing.  Maybe have this above the options
```ruby
FoundationRailsHelper.configure do |config|
  # your options here
end
```
then show something like
```ruby
config.ignored_flash_keys
```
below the option's description.